res/i18n/fr/actions.json

```diff
@@ -61,6 +62,7 @@
 	"profile": "Profil",
 	
 	"reset": "Réinitialiser",
+	"run_stop": "Exécuter/arrêter",
```

Comment:
Is it possible to shorten words to 3-4 symbols per each?

Reply:
done